allauth/account/adapter.py

```diff
@@ -464,7 +464,7 @@ def respond_email_verification_sent(self, request, user):
 
     def _get_login_attempts_cache_key(self, request, **credentials):
         site = get_current_site(request)
-        login = credentials.get('email', credentials.get('username', ''))
+        login = credentials.get('email', credentials.get('username', '')).upper()
```

Comment:
This should only be case-insensitive when using the email address for the key, not the username.